pytorch_lightning/callbacks/model_checkpoint.py

```diff
@@ -140,6 +140,7 @@ class ModelCheckpoint(Callback):
 
     CHECKPOINT_JOIN_CHAR = "-"
     CHECKPOINT_NAME_LAST = "last"
+    FILE_EXTENSION = "ckpt"
```

Comment:
```suggestion
    FILE_EXTENSION = ".ckpt"
```
lets keep the `.` as `os.path.extsplit(...)` holds it too